### ISSUE TITLE
Unable to regain keyboard focus after clicking away in Renoise (VST)

### ISSUE THREAD

**Issue by bigdaddyben** [DISTRHO/Cardinal#234]
Upon first loading Renoise and entering the VST window, keyboard input works in Cardinal, however upon clicking away and using the keyboard in the DAW, the plugin will stop responding to keyboard inputs. This may be preceded by a short period of laggy/intermittent response before it stops responding entirely. The "enable keyboard" checkbox at the bottom of the plugin window successfully stops the keyboard signals going to the DAW but has no effect on the responsiveness of the plugin.

**Comment by falkTX** [comment]
Which OS?

**Comment by bigdaddyben** [comment]
Hi, Windows 10

**Comment by falkTX** [comment]
I am closing this ticket in favour of a global one for tracking keyboard input behaviour under many hosts and OSes.
See #304
Discussion should now go there.
